chain/bitcoind_conn.go

```diff
@@ -23,6 +23,10 @@ const (
 	// notifications from bitcoind through ZMQ.
 	rawTxZMQCommand = "rawtx"
 
+	// sequenceZMQCommand is the command used to receive sequence
+	// notifications from bitcoind through ZMQ.
+	sequenceZMQCommand = "sequence"
```

Comment:
IIUC, this is only in _newer_ versions of `bitcoind`, so we'd need to do a version check if we were trying to enable it in a graceful manner.

Reply:
Removed, will add the check if we ever decide to implement it.